Resources/Prototypes/_White/Entities/Clothing/Hands/gloves.yml

```
@@ -19,3 +19,25 @@
     sprite: _White/Clothing/Hands/sheriff_gloves.rsi
   - type: Clothing
     sprite: _White/Clothing/Hands/sheriff_gloves.rsi
+
+- type: entity
+  parent: ClothingHandsGlovesLatex
+  id: ClothingHandsGlovesVinilDark
+  name: dark vinyl gloves
+  description: Dark vinyl gloves that make your hands sweat even more than latex and nitrile gloves! But fortunately, due to the hygiene issues of most crew members, no one will notice. Especially if you're a synthetic.
+  components:
+  - type: Sprite
+    sprite: _White/Clothing/Hands/dark_vinile_gloves.rsi
+  - type: Clothing
+    sprite: _White/Clothing/Hands/dark_vinile_gloves.rsi
```

```suggestion
    sprite: _White/Clothing/Hands/dark_vinil_gloves.rsi
  - type: Clothing
    sprite: _White/Clothing/Hands/dark_vinil_gloves.rsi
```
Пути надо еще переименовать